package/main/daml/Daml.Finance.Instrument.Bond/daml.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
   - daml-stdlib
 data-dependencies:
   - .lib/contingent-claims/v3.0.0.20220721.1/contingent-claims-3.0.0.20220721.1.dar
+  - .lib/daml-finance/Daml.Finance.Instrument.Generic/0.1.4/daml-finance-instrument-generic-0.1.4.dar
```

Comment:
although it doesn't matter, I think it is good practise to keep the list in the same order as we build them